packages/graphql-auth-transformer/src/ModelAuthTransformer.ts

```diff
@@ -1287,8 +1291,10 @@ All @auth directives used on field definitions are performed when the field is r
                 )
 
                 const collectAuthCondition = this.resources.collectAuthCondition()
+                const staticGroupAuthorizedVariable = field ? `${field.name.value}_${ResourceConstants.SNIPPETS.IsStaticGroupAuthorizedVariable}` :
```

Comment:
leftover? can we use the helper method here?

Reply:
removed leftover